nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -4,6 +4,8 @@
 with lib;
 
 let
+  json = pkgs.formats.json {};
+  pipewire-json-conf = pkgs.callPackage ./pipewire-json-config.nix { pipewire = cfg.package; };
```

Comment:
Is there a reason you are using mkDerivation here instead of i.e. pkgs.runCommand?